lib/shopify_app/shop.rb

```diff
@@ -4,12 +4,20 @@ module Shop
 
     included do
       validates :shopify_domain, presence: true, uniqueness: true
-      validates :shopify_token, presence: true
+      if shopify_token_needs_to_exist?
```

Comment:
Any reason to not just use
```ruby
validates :shopify_token, presence: true, if: :shopify_token_needs_to_exist?
```

also, `shopify_token_required?` feels less awkward to me, but naming things is hard 😇